utilities/src/main/java/ru/study21/jcsv/xxl/algorithms/MyersDiff.java

```diff
@@ -0,0 +1,226 @@
+package ru.study21.jcsv.xxl.algorithms;
+
+import ru.study21.jcsv.xxl.algorithms.SortDescription.KeyType;
+import ru.study21.jcsv.xxl.data.CSVTable;
+
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class MyersDiff {
+
+    private List<Point> _shortestPath;
+    private final List<KeyInfo> _keys;
+    private List<Operation> _operations;
+
+    public MyersDiff(List<KeyInfo> _keys) {
+        this._keys = _keys;
+    }
+
+    public static record KeyInfo(int field, KeyType keyType) {
+    }
+
+    public static record Point(int x, int y) {
+    }
+
+    public static record Operation(OperationType type, String content) {
+    }
```

Comment:
Do we really need all these  records be public ?

Reply:
Corrected